inc/classes/dependencies/mobiledetect/mobiledetectlib/Mobile_Detect.php

```diff
@@ -216,1477 +216,1477 @@
  */
 class WP_Rocket_Mobile_Detect
 {
-    /**
```

Comment:
this is dependency update that we don't need here

Reply:
Yes, I know, I prefer not touching any file other than needed files, I pushed a small commit to revert this.
Thanks Boss.